lib/util/index.coffee

```diff
@@ -39,6 +40,26 @@ exports.treatAsMissingDevice = (uuidOrId) ->
 		replacementErr.stack = err.stack
 		throw replacementErr
 
+exports.osVersionCompare = (versionA, versionB) ->
```

Comment:
Also NIT: this is R-compare so I would rename the method to make it obvious the comparison is inversed here.